rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp

```diff
@@ -81,7 +81,70 @@ class MultiThreadedExecutor : public rclcpp::Executor
 private:
   RCLCPP_DISABLE_COPY(MultiThreadedExecutor)
 
-  std::mutex wait_mutex_;
+  /// \internal A mutex that has two locking mechanism, one with higher priority than the other.
+  /**
+   * After the current mutex owner release the lock, a thread that used the high
+   * priority mechanism will have priority over threads that used the low priority mechanism.
+   */
+  class MutexTwoPriorities
+  {
+public:
+    class HpMutex
```

Comment:
Let's avoid abbreviations, I assume this should be `HighPriorityMutex` and `LpMutext` should be `LowPriorityMutex`?

Reply:
And below, avoid things like `lp` and `hp` and `low_prio_` and `lpg`.

Reply:
Done in https://github.com/ros2/rclcpp/pull/1516/commits/7b9548d583f9f822809dc0d8b12fa60997b1e2e9.